### PR TITLE
don't copy config if to and from projects are the same

### PR DESCRIPTION
**Pull request by chrisamiller**

### CODE REVIEW

**Comment by chrisamiller** [comment]
Guess how i figured out you could do this... :rage3:

File: lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.pm [hunk 1]
```diff
@@ -50,6 +50,11 @@ sub execute {
     my $from_project = $self->from_project;
     my $to_project = $self->to_project;
 
+    if($from_project->id eq $to_project->id){
+        $self->error_message("To and From projects cannot be the same");
+        return 0;
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.pm]: I think we usually `return;` rather than `return 0;` in this circumstance.

**Comment by tmooney** [comment]
+1, yeah, that's a bummer!

**Comment by chrisamiller** [comment]
Sure thing - updated.

**Comment by brummett** [comment]
+1

**Pull request merged**